src/sklearn_evaluation/tracker.py

```diff
@@ -186,6 +187,9 @@ def __getitem__(self, key):
         return self._data[key]
 
 
+tel_tracker = telemetry.create_group("sqlitetracker")
```

Comment:
Again, we're changing SQLiteTracker to sqlitetracker, we might see a big hit on the data unless we change it in both ends.